twitter.gt2eb.user.js

```diff
@@ -372,6 +372,7 @@
     hideWhoToFollow:          false,
     hideTranslateTweetButton: false,
     hideMessageBox:           true,
+    enableQuickBlock:         true,
```

Comment:
```suggestion
    enableQuickBlock:         false,
```
On second thought, diabling it by default is even better.

Reply:
Thank you for the suggestion and I'm glad to help! If there is also a better place in the settings menu to put the toggle, let me know.

Reply:
I think the "Other" section is fine